Facebook_Video_downloader/script.py

```diff
@@ -0,0 +1,12 @@
+import re
+import requests
+import urllib.request
+from tqdm import tqdm
+
+video_link = input("Enter the video link: ")
+
+req = requests.get(video_link, stream=True)
+
+print("downloading......")
```

Comment:
This is not working, I tried with this link: https://www.facebook.com/herowarsgame/videos/961066781305631/

A file was downloaded but it's not playing. Also, as mentioned in the issue description, implement the GUI and then we will see if progress bar can be implemented